README.md

```diff
@@ -1048,6 +1048,7 @@ Table of Contents
   * [Play with Docker](https://labs.play-with-docker.com/) — A simple, interactive and fun playground to learn Docker.
   * [quay.io](https://quay.io/) — Build and store container images with unlimited free public repositories
   * [Whales](https://github.com/Gueils/whales) — A tool to automatically dockerize your applications for free.
+  * [Portainer](https://www.portainer.io/) — An open source container management tool for Kubernetes, Docker, Docker Swarm and Azure ACI.
```

Comment:
Please note this list is for SaaS software, that's software delivered as a service not self hosted or open source projects you run on your own machines

Reply:
noted thanks 👍🏾 I will take this PR down

Reply:
The main reason I put this one up is that in dev-ops it can be very useful to monitor nodes DB etc